e2e/paths/enterOrdersAndDirections_test.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../config.js');
+
+Feature('Enter order and details');
+
+Before((I, caseViewPage) => {
+  I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventSummary);
+  caseViewPage.goToNewActions(config.applicationActions.selectOrders);
+});
+
+Scenario('Select care order', 
+(I, ordersNeededPage) => {
+  ordersNeededPage.SelectCareOrderOnly();
+  I.continueAndSubmit(config.eventSummary, config.eventDescription);
+  I.seeEventSubmissionConfirmation(config.applicationActions.selectOrders);
+});
+
+Scenario('Apply for all case orders and add directions and interim orders', 
+  (I, ordersNeededPage) => {
```

Comment:
same here

Reply:
At what point do we break line? In my opinion, readability is improved here with the line break.